### ISSUE TITLE
README documentation needs updating

### ISSUE THREAD

**Issue by caldwellst** [OCHA-DAP/pa-aa-toolbox#34]
The README documentation on the develop page needs updating. It currently references `python setup.py ...` in multiple instances even though the `setup.py` file is no longer being used.

On a similar note, running `pre-commit run --all-files` produces empty documentation for certain modules, such as everything in `utils`.

Manually specifying the rebuild using the below code works in generating the documentation for those modules:

```shell
sphinx-build -b html -d docs/build/doctrees docs/source docs/build/html
```

I am not very familiar with the setup we are using now and what's ideal, so raising this issue so others can input.

**Comment by turnerm** [comment]
Oops sorry, this definitely my bad, will look into it!

**Comment by turnerm** [comment]
@caldwellst I've checked and as far as I can tell the hook is running the command you specified above. To be honest it wasn't a crucial hook plus it doesn't appear to be widely used, so I've removed it in #35. 

However, I don't find any references to `setup.py` in the main README -- were you perhaps looking at the `main` branch? I can set `develop` to be the frontpage branch in case that's where the confusion comes from.
